src/ActionsTests/Misc/ExtensionMethodsTests.cs

```diff
@@ -21,19 +20,16 @@ public class ExtensionMethodsTests
         [TestMethod()]
         public void GetSmallestElementFromPointTest()
```

Comment:
The changes in this file are much clearer if you enable "Hide whitespace changes". We're just removing unnecessary `using ShimsContext.Create())` blocks